### ISSUE TITLE
Allow contributors to self-assign issues with GitHub workflow

### ISSUE THREAD

**Issue by sugatoray** [JohnSnowLabs/nlptest#319]
Commenting `#self-assign` in any issue will let repository members to self-assign that issue to themselves.

:point_right: This reduces chores for repo-maintainer(s).

**Comment by sugatoray** [comment]
cc: @ArshaanNazir

I have already sent a PR for adding this functionality. Please assign this to me. 

> 🔔 If/once the PR is merged, then in future, issue assignment can be done by contributors themselves with `#self-assign` comment alone.

- PR: #320  

> 👉 Like this: **#self-assign**

**NOTE**: 

*I have also raised another issue #318 to add `nlptest` to conda-forge channel and enable conda driven installation. I was not able to self-assign then. I have earlier added this workflow to other repositories as well. Please see the following PR:* 

- https://github.com/unionai-oss/unionml/pull/136
